webapp/app/js/controllers/main.js

```diff
@@ -109,6 +110,13 @@ treeherder.controller('MainCtrl', [
                 $rootScope.$emit(thEvents.selectPreviousUnclassifiedFailure);
             });
 
+            // Shortcut: retrigger selected job
+            Mousetrap.bind('r', function() {
+                $scope.$evalAsync(
+                    $rootScope.$emit(thEvents.jobRetrigger, $rootScope.selectedJob)
```

Comment:
Totally correct. Ed you are awesome, thanks for catching it. We should always test for selected job at shortcut invocation when applicable. I've switched it and it seems to be working fine in Firefox and Chrome and with other shortcut integration testing (related bugs, comment, save, clear, pin). I'll push the delta, then squash after you've had a look.